spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/LibraryControllerHelperService.java

```diff
@@ -673,6 +674,12 @@ public JSONObject addLibraryQC(HttpSession session, JSONObject json) {
     return JSONUtils.SimpleJSONError("Cannot add LibraryQC");
   }
 
+  private double round(double value, int numberOfDigitsAfterDecimalPoint) {
```

Comment:
It might be useful to make this `public` and `static` and move it to `/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/LimsUtils.java`.

I find the whole spring module thing a little odd...